src/datepicker/calendar-header.js

```diff
@@ -49,49 +52,79 @@ export default class CalendarHeader extends React.Component<HeaderPropsT> {
     this.props.onMonthChange({date: subMonths(this.props.date, 1)});
   };
 
+  setActive = () => {
+    this.props.setActiveState(true);
+  };
+
+  setInactive = () => {
+    this.props.setActiveState(false);
+  };
+
   renderPreviousMonthButton = () => {
-    const allPrevDaysDisabled = monthDisabledBefore(
-      this.props.date,
-      this.props,
+    const {date, overrides = {}} = this.props;
+    const allPrevDaysDisabled = monthDisabledBefore(date, this.props);
+    const [PrevButton, prevButtonProps] = getOverrides(
+      overrides.PrevButton,
+      ArrowLeft,
     );
-    if (allPrevDaysDisabled) {
-      return;
-    }
```

Comment:
Do we actually want avoid rendering prev and next buttons when all dates before/after are disabled?

Reply:
i think so, it confirms that disabled dates are not selectable rather than a rendering bug

but paging @nguyensomniac on it

Reply:
reverted it